### ISSUE TITLE
Samsung UN60H6350 - KEY_POWER bug?

### ISSUE THREAD

**Issue by cramerry** [kkapitan/homebridge-homesung#14]
I have a Samsung UN60H6350 and have gotten some *some* of the functions working via Homebridge (i.e., volume control), but I have not been able to successfully get the KEY_POWER key to work.  I have no errors in the log aside from when I attempt to use the KEY_POWER key set up to power off the TV after turning it on with my remote, and the logs indicate it "does not support powering on."  Below are the "power" stanza from my config.json file, as well as a paste of my log output.  Any thoughts as to what may be preventing me from using this?  Is it simply not accessible for my model TV?

**excerpt from config.json:**
`  "power": {
                "name": "TV KEY_POWER",
                "key": "KEY_POWER",
                "enableCEC": false
              },`

**Log output:**
2019-01-28 22:52:51,stdout,"[37m[1/28/2019, 10:52:51 PM][39m [36m[SamsungTV][39m Power via WIFI does not support powering on

2019-01-28 22:52:35,stdout,"[37m[1/28/2019, 10:52:35 PM][39m [36m[SamsungTV][39m TV status: standby

2019-01-28 22:51:36,stdout,"[37m[1/28/2019, 10:51:36 PM][39m Homebridge is running on port 51826.

**Comment by skop1986** [comment]
I have such a problem too! Check when the TV is off it is disconnected from the Internet?

**Comment by cramerry** [comment]
Yes, this model won’t be connected once it is off; I’m trying to enable a power off when it is on, however.

**Comment by locnho** [comment]
Hi,

I just want to share that I initial used this and got this all working but limited. But at the end, I got a Broadlink mini rm3 with homebridge instead. Then use the Broadlink mini rm3 with an IR Remote Control Extender/IR Repeater. With that, it is much more supported and works great.
